app/views/declaration/summary/notifications_section.scala.html

```diff
@@ -37,9 +38,9 @@
         ))
         ++ notifications.map(notification =>
             SummaryListRow(
-                classes = s"${notification.displayStatus.toLowerCase}-row",
```

Comment:
It seems to me that this was wrong in the former implementation as we would end up with class names that have spaces between the words. Please let me know in case I'm wrong.

Reply:
:+1: